back-end/persistence/texts.js

```diff
@@ -0,0 +1,24 @@
+const router = require("express").Router();
+let Text = require("./models/texts.model");
+
+router.route("/").get((req, res) => {
+	Text.find()
+		.then((texts) => res.json(texts))
+		.catch((err) => res.status(400).json("Error: " + err));
+});
+
+router.route("/add").post((req, res) => {
+	// const name = req.body.name;
+	const text = req.body.text;
+	const source = req.body.source;
+	const creationDate = req.body.creationDate;
+
+	const newText = new Text({ text, source, creationDate });
```

Comment:
In the future, we might wanna add the update and delete date when posting a new object. The value will be set to null but at least the attribute will be in the table already.